dev.yml

```diff
@@ -0,0 +1,26 @@
+name: identity_cache
+
+up:
+  - homebrew:
+    - postgresql
+  - ruby:
+      version: 2.2.3p172-shopify
+      package: shopify/shopify/shopify-ruby
+  - railgun
+  - bundler
+
+commands:
+  test:
+    run: MYSQL_HOST=192.168.64.94 POSTGRES_HOST=192.168.64.94 MEMCACHED_HOST=192.168.64.94 bundle exec rake test
+
+railgun:
+  image: dev:railgun-common-services-0.2.x
+  ip_address: 192.168.64.94
```

Comment:
this looks like it's taken by hubspot-dashboards. You'll have to pull first (I should make this work better). The next available IP is `.98`.